contracts/src/token/erc721/mod.rs

```diff
@@ -662,9 +660,6 @@ impl Erc721 {
     /// * `&mut self` - Write access to the contract's state.
     /// * `account` - Account to increase balance.
     /// * `value` - The number of tokens to increase balance.
-    // TODO: Right now this function is pointless since it is not used.
-    // But once we will be able to override internal functions,
-    // it will make a difference.
```

Comment:
This is probably the one that needs more discussion, but:
A) Internal function overriding is not very "rusty", so probably won't happen?
B) It is still useful for people writing extensions and being able to call this right? I think that's actually the way it is designed to work in Solidity

Reply:
A) 💯 
B) I cannot see big value in this function as it does not provide anything special but an additional "setter". IMHO it is fine to remove this fn.

Reply:
Basically it is used at consecutive now. In solidity it is impossible to access private fields from child contracts. At stylus it can work, but I feel it's a bad development practice.
As I see the function is just for convenience.
We can remove misleading `TODO`.